### PR TITLE
Ignore some tests on aarch64 to pass the run-pass test on aarch64-linux-android

### LINKED ISSUE #21773: "make check" doesn't work correctly on aarch64-linux-android
Not enough support for aarch64-linux-android tests.

### PR DESCRIPTION
**Pull request by sae-bom**
Resolved #21773. (Aarch64 test has been broken again)
r? @alexcrichton

### CODE REVIEW

**Comment by sae-bom** [comment]
cc @yichoi

File: src/compiletest/util.rs [hunk 1]
```diff
@@ -25,6 +25,23 @@ const OS_TABLE: &'static [(&'static str, &'static str)] = &[
...
+    ("sparc", "Sparc"),
+    ("powerpc", "PowerPC"),
+    ("arm64", "AArch64"),
+    ("arm", "ARM"),
+    ("aarch64", "AArch64"),
+    ("mips", "Mips"),
+    ("xcore", "XCore"),
+    ("msp430", "MSP430"),
+    ("hexagon", "Hexagon"),
+    ("s390x", "SystemZ"),
```
Comment (alexcrichton) [inline src/compiletest/util.rs]: Most of our ignore directives are all-lowercase, could these map to `ignore-aarch64` for example?

**Comment by alexcrichton** [comment]
Looks good to me! Just one minor nit and otherwise r=me

**Comment by alexcrichton** [comment]
@bors: r+ 

Thanks!

**Pull request merged**